content/apps/hell.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Dynamic Links: Hell AP
+parent: Applications
```

Comment:
The Applications section starts with:
```
This section describes sample applications with SCION support. These applications are maintained in our scion-apps GitHub repository.
```
This does not really match. Admittedly, RAINS is already a bit borderline. But the Hell AP seems like a different category. To be honest, I don't know a good title for a section where this would fit. Something like "SCIONLab network features", but that's not very catchy and overly generic.

Reply:
Sometimes this style of artificial degradation is filed under "Chaos Engineering", not sure if that is applicable here.

Reply:
Landing:
>It also provides a list of interesting projects that are using the SCION infrastructure for communication.

We could also simply have a category project. I'll leave reorganizing the docs structure for a separate PR.
Having it in the apps section kinda fits for the time being since you'd use it with exactly those apps and the config is already shared as part of the post (no separate repo where there is something to publish).